<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1113</width>
                <height>527</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <item>
                                        <widget class="QPushButton" name="QuoteTextInputButton">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>PushButton</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="QuoteTextOutputButton">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                                    <horstretch>120</horstretch>
                                                    <verstretch>45</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>PushButton</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QCheckBox" name="QuoteEvalCheckBox">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>CheckBox</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="QuoteEncodeButton">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>PushButton</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="QuoteTextBox"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QPushButton" name="QuoteCipherInputButton">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>PushButton</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="QuoteCipherOutputButton">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>PushButton</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer_2">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="QuoteDecodeButton">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>120</width>
                                                    <height>45</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>PushButton</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="QuoteCipherBox"/>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
